pythran/pythonic/utils/seq.hpp

```diff
@@ -20,7 +20,7 @@ namespace pythonic {
             struct rgens : rgens<N-1, sizeof...(S), S...> { };
 
         template<int ...S>
-            struct rgens<1, S...> {
+            struct rgens<0, S...> {
```

Comment:
It should not be the sizeof...(S) + 1 instead? To be consistent with gens?
It looks like gens<5> => gens<4, 4> => gens<3, 3, 4> => gens<2, 2, 3, 4> => gens<1, 1, 2, 3, 4> => seq<1, 2, 3, 4>
whil rgens<5> => rgens<4, 0> => rgens<3, 1, 0> => rgens<2, 2, 1, 0> => rgens<1, 3, 2, 1, 0> => rgens<0, 4, 3, 2, 1, 0> => seq<4, 3, 2, 1, 0>
with sizeof...(S) -1, you will have seq<4, 3, 2, 1>

BTW, I have to understand this each time I come here. Can you just add a comment about what is happening? 
SOmethink like:
```
rgens<S> => seq<S-1, ..., 1>
gens<S> => seq<1, ..., S - 1>
```

If you prefer to keep it that way, it is not a problem to me but please add a comment about the values in the sequence.

Also, I wonder if gens<S> should not return seq<0, S-1>? It would avoid the +1, -1 when we use it.
Or may be we should just switch to use C++14 : http://en.cppreference.com/w/cpp/utility/integer_sequence